security/tasks.py

```diff
@@ -208,6 +212,11 @@ def on_retry_task(self, task_run_log, args, kwargs, exc, eta):
             output=self.request.output_stream.getvalue(),
             estimated_time_of_next_retry=eta
         )
+        task_log = task_run_log.get_task_log()
+        if task_log:
+            task_log.change_and_save(
+                state=CeleryTaskLogState.RETRIED
+            )
```

Comment:
we can set task_log state if is alredy stored in the database (with non atomic commection the task will be stored always)